fala/apps/adviser/views.py

```diff
@@ -4,11 +4,37 @@
 from django.conf import settings
 from django.urls import resolve, reverse
 from django.views.generic import TemplateView, ListView
+from django.http import HttpResponse
+import os
 
 from .forms import AdviserSearchForm, AdviserRootForm
 from .laa_laa_paginator import LaaLaaPaginator
 from .regions import Region
 
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
should this be in settings.py?

Reply:
Yes, this should be in settings / base.py 

So you can do something like this `settings.BASE_DIR` or whatever to access the BASE_DIR variable